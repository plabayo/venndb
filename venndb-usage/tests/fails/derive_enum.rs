use venndb::VennDB;

#[derive(VennDB)]
enum MyEnum {
    A,
    B,
}

fn main() {}
