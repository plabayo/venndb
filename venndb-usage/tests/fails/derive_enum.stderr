error: Only Structs with named fields are supported
 --> tests/fails/derive_enum.rs:4:1
  |
4 | / enum MyEnum {
5 | |     A,
6 | |     B,
7 | | }
  | |_^
